src/pages/ProductsListing.jsx

```diff
@@ -1,7 +1,60 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Filter, FilteredProducts } from "../component/index";
+import { useProducts } from "../state";
+import { Audio } from "react-loader-spinner";
+import "../styles/Filter.css";
 
 const ProductsListing = () => {
-  return <div>This is Product Listing Page</div>;
+  const {
+    productsState: { loading, error },
+  } = useProducts();
+  const [displayMobileFilter, setMobileFilter] = useState(false);
+
+  useEffect(() => {
+    console.log(window.innerWidth, "width");
+    window.addEventListener("resize", () => {
+      if (window.innerWidth > 1000) {
+        setMobileFilter(true);
+      } else {
+        setMobileFilter(false);
+      }
+    });
+    if (window.innerWidth > 100) {
+      setMobileFilter(true);
+    }
+    return () => {
+      window.removeEventListener("resize");
+    };
```

Comment:
Nice! 
A small suggestion (you need not do this now) but you can also create a separate custom hook and then whenever you need sidebars/ hamburger menus in smaller screens use this!